src/style.css

```diff
@@ -42,7 +42,69 @@
 }
 
 .main-container {
-  height: 80vh;
+  flex-direction: column;
+  min-height: 600px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.list-movies {
+  width: 90%;
+  list-style: none;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
```

Comment:
great way to display your elements always Keep It Simple 👍 keep it up, using short lines of code makes your work neat and your program run faster 🥳

Reply:
Thank you, I made the necessary change 👍